bundles/org.openhab.core.config.discovery.addon/src/main/java/org/openhab/core/config/discovery/addon/AddonSuggestionService.java

```diff
@@ -80,7 +80,7 @@ public AddonSuggestionService(final @Reference ConfigurationAdmin configurationA
         this.configurationAdmin = configurationAdmin;
         this.localeProvider = localeProvider;
 
-        SUGGESTION_FINDERS.forEach(f -> baseFinderConfig.put(f, true));
+        SUGGESTION_FINDERS.forEach(f -> baseFinderConfig.put(f, false));
         modified(config);
 
         // Changes to the configuration are expected to call the {@link modified} method. This works well when running
```

Comment:
Side question: Is this job still needed after #4188?

Reply:
It is. The reason is that it refers to a configuration from a different component (to avoid proliferation of component configurations all separated out in the UI) and in Karaf that does not get synchronized reliably.